src/main/java/com/locallampoon/fiveh/core/Game.java

```diff
@@ -7,14 +7,15 @@
 import java.util.Map;
 
 public class Game implements Serializable {
-    private final Player player;
-    private Map<String, Room> houseMap;
+    private static Player player;
```

Comment:
For my own personal edification, does making these variables static serve a functional purpose? Like you need to load them before anything else in the class is run/initialized?

Reply:
The intent to make these static was for accessibility outside the Game class without the need to creating a new instance. The ConsolePanel calls Game.handleCommand(), which any dependent variables need to be static in order to be accessed.